src/dmd/link.d

```diff
@@ -152,6 +152,48 @@ version (Posix)
     }
 }
 
+version (Windows)
+{
+    private void writeQuotedArgIfNeeded(ref OutBuffer buffer, const(char)* arg)
+    {
+        bool quote = false;
+        for (size_t i = 0; arg[i]; ++i)
+        {
+            if (arg[i] == '"')
+            {
+                quote = false;
+                break;
```

Comment:
This now assumes that if the command line already contained quotes (after C's interpretation) the user knows what he's been doing. I'm ok with that (but expected the check to actually look for spaces not inside a pair of quotes).